webapp/channels/src/components/admin_console/message_export_settings.tsx

```diff
@@ -51,13 +50,16 @@ const messages = defineMessages({
     exportJobStartTime_title: {id: 'admin.complianceExport.exportJobStartTime.title', defaultMessage: 'Compliance Export Time:'},
     exportJobStartTime_description: {id: 'admin.complianceExport.exportJobStartTime.description', defaultMessage: 'Set the start time of the daily scheduled compliance export job. Choose a time when fewer people are using your system. Must be a 24-hour time stamp in the form HH:MM.'},
     exportFormat_title: {id: 'admin.complianceExport.exportFormat.title', defaultMessage: 'Export Format:'},
-    exportFormat_description: {id: 'admin.complianceExport.exportFormat.description', defaultMessage: 'Format of the compliance export. Corresponds to the system that you want to import the data into.{lineBreak} {lineBreak}For Actiance XML, compliance export files are written to the exports subdirectory of the configured [Local Storage Directory]({url}). For Global Relay EML, they are emailed to the configured email address.'},
+    exportFormat_description: {
+        id: 'admin.complianceExport.exportFormatDetail.description',
+        defaultMessage: 'Format of the compliance export. Corresponds to the system that you want to import the data into.{br}{br}For Actiance XML, compliance export files are written to the exports subdirectory of the configured <a>Local Storage Directory</a>. For Global Relay EML, they are emailed to the configured email address.',
+    },
```

Comment:
could you please bring it down in the FormattedMessage itself for clarity?

Reply:
@M-ZubairAhmed, I wanted to point out that the exportFormat_description is currently part of the array of searchable strings at line 64. If we move it down into its FormattedMessage, the searchable strings will no longer have access to it.

Since the searchable string array is utilized in admin_definition.tsx, could you please advise on how we should proceed with this?

Reply:
oh i see then its fine please ignore my comment